src/app/collective-rewards/allocations/context/AllocationsContext.tsx

```diff
@@ -276,14 +290,14 @@ export const AllocationsContextProvider: FC<{ children: ReactNode }> = ({ childr
 }
 
 function createInitialAllocations(
-  rawAllocations: bigint[],
+  rawAllocations: (bigint | undefined)[],
   rawBuilders: Builder[],
   selections: State['selections'],
 ): [Allocations, bigint, number] {
   return rawAllocations.reduce(
     (acc, allocation, index) => {
       const builderAddress = rawBuilders[index].address
-      if (allocation > 0n || selections[builderAddress]) {
+      if (allocation && (allocation > 0n || selections[builderAddress])) {
```

Comment:
This is not the same condition, by requiring allocation, you are nullifying selections, since selections does not have allocations, since it's just a boolean map.

Reply:
You're right, well spotted! The parentheses are in the wrong place. But actually, I will just ensure there is a value.

Thank you!